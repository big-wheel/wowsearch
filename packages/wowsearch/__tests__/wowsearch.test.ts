/**
 * @file main.test
 * @author Cuttle Cong
 * @date 2018/6/10
 * @description
 */
import { readFileSync } from 'fs'
import { inspect } from 'util'
import * as readJSONSync from 'read-json-sync'
import { makeFixture } from './help'
import { default as wowsearch, getUrlList } from '../index'

describe('wowsearch', function() {
  it.skip('getUrlList', async function() {
    const urls = await getUrlList(
      {
        selectors: {},
        start_urls_patterns: [/.*/],
        sitemap_urls: ['https://imcuttle.github.io/sitemap.xml']
      }
    )
    expect(urls.map(x => x.url)).toMatchSnapshot()
  })
})
