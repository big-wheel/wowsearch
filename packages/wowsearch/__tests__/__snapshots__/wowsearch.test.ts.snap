// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`wowsearch getUrlList 1`] = `
Array [
  "https://imcuttle.github.io/Moka-birth",
  "https://imcuttle.github.io/Set-Up-Yourself-Blog",
  "https://imcuttle.github.io/about-promise-and-qjs",
  "https://imcuttle.github.io/about-snap-fade-away",
  "https://imcuttle.github.io/advanced-bash-scripting-guide-1",
  "https://imcuttle.github.io/ali-app-pay",
  "https://imcuttle.github.io/animation-of-my-blog-navbar",
  "https://imcuttle.github.io/b_vs_strong_&_i_vs_em",
  "https://imcuttle.github.io/baidu-fe-interview-1",
  "https://imcuttle.github.io/baidu-fe-interview-2",
  "https://imcuttle.github.io/birdv3-standup",
  "https://imcuttle.github.io/blog-update",
  "https://imcuttle.github.io/bookshop",
  "https://imcuttle.github.io/componentWillReceiveProps-vs-componentWillUpdate",
  "https://imcuttle.github.io/deploy-demo-page-automatically",
  "https://imcuttle.github.io/detect-tree-changed",
  "https://imcuttle.github.io/dev-babel-plugin",
  "https://imcuttle.github.io/diy-一个-commonjs-运行环境吧",
  "https://imcuttle.github.io/doc-import-from-markdown",
  "https://imcuttle.github.io/doc-pipe-boom",
  "https://imcuttle.github.io/draw-flower-by-canvas",
  "https://imcuttle.github.io/easy-img-viewer",
  "https://imcuttle.github.io/express4x-file-explorer",
  "https://imcuttle.github.io/feel-hexo-power",
  "https://imcuttle.github.io/from-timer-to-webworker",
  "https://imcuttle.github.io/funny-something-summary",
  "https://imcuttle.github.io/gp-tranaltion",
  "https://imcuttle.github.io/graceful-document-maker-picidae",
  "https://imcuttle.github.io/graduation-weekly-record-3",
  "https://imcuttle.github.io/guide-of-gojs",
  "https://imcuttle.github.io/h5-offline-way",
  "https://imcuttle.github.io/hello-moka-desktop",
  "https://imcuttle.github.io/home-pics",
  "https://imcuttle.github.io/how-to-record-command-line",
  "https://imcuttle.github.io/how-to-write-babel-plugin-danger-remove-unused-import",
  "https://imcuttle.github.io/how-to-write-component-demo",
  "https://imcuttle.github.io/html-to-markdown-by-javascript",
  "https://imcuttle.github.io/i18n-live-edit",
  "https://imcuttle.github.io/img-normalize",
  "https://imcuttle.github.io/img-recover",
  "https://imcuttle.github.io/imgs-fallwater-by-js",
  "https://imcuttle.github.io/instant-git-test",
  "https://imcuttle.github.io/linux-c-osx-sendfile",
  "https://imcuttle.github.io/linuxc-learning-one-week",
  "https://imcuttle.github.io/make-dns-http",
  "https://imcuttle.github.io/make-psd-measurable",
  "https://imcuttle.github.io/make-vlan",
  "https://imcuttle.github.io/markdown-syntax-test",
  "https://imcuttle.github.io/min-function-optimization",
  "https://imcuttle.github.io/mobx-url-sync-decorator",
  "https://imcuttle.github.io/mobx-vs-redux-with-react",
  "https://imcuttle.github.io/moka-desktop-program-record",
  "https://imcuttle.github.io/moka-hooks",
  "https://imcuttle.github.io/my-projects",
  "https://imcuttle.github.io/never-simple-student",
  "https://imcuttle.github.io/nginx-reverse-proxy",
  "https://imcuttle.github.io/njnu-stuents-faces-recognition",
  "https://imcuttle.github.io/node-express-jade-http-file-explorer",
  "https://imcuttle.github.io/node-ftp-client",
  "https://imcuttle.github.io/node-js-前后端分离开发新思路",
  "https://imcuttle.github.io/node-smtp",
  "https://imcuttle.github.io/npm-cached-option",
  "https://imcuttle.github.io/number-code-verify",
  "https://imcuttle.github.io/o(nd)-difference-algorithm",
  "https://imcuttle.github.io/page-mark",
  "https://imcuttle.github.io/paper-of-graduatio-project",
  "https://imcuttle.github.io/play_css3_Animation",
  "https://imcuttle.github.io/pr-of-github",
  "https://imcuttle.github.io/react-data-binding-based-on-mobx",
  "https://imcuttle.github.io/react-mobx-vm",
  "https://imcuttle.github.io/react-native-starter",
  "https://imcuttle.github.io/record-once-javascript-regexp-landfill",
  "https://imcuttle.github.io/repost-progressive-web-app",
  "https://imcuttle.github.io/request-song-robot",
  "https://imcuttle.github.io/requestanimationframe-vs-setinterval",
  "https://imcuttle.github.io/resolution-of-react-component-demo-page",
  "https://imcuttle.github.io/say-goodbye-to-manual-refresh",
  "https://imcuttle.github.io/search-function-for-blog",
  "https://imcuttle.github.io/segumentfault-markdown-editor-and-mo-editor",
  "https://imcuttle.github.io/share-font-awesome",
  "https://imcuttle.github.io/simple-imgprocessing",
  "https://imcuttle.github.io/slice-and-breakpoint-up-down",
  "https://imcuttle.github.io/some-configuration-files-of-nodejs-project",
  "https://imcuttle.github.io/some-fegl-programs",
  "https://imcuttle.github.io/some-words-for-students",
  "https://imcuttle.github.io/spider-girls-bynode",
  "https://imcuttle.github.io/sync-request-in-nodejs",
  "https://imcuttle.github.io/talk-about-javascript-object-and-array-deepclone",
  "https://imcuttle.github.io/talk-about-live-markd",
  "https://imcuttle.github.io/talk-about-react-hoc",
  "https://imcuttle.github.io/talk-about-websocket",
  "https://imcuttle.github.io/telescopejs",
  "https://imcuttle.github.io/the-core-algorithm-of-doc-md-import",
  "https://imcuttle.github.io/the-mena-of-university",
  "https://imcuttle.github.io/the-review-of-spartacus",
  "https://imcuttle.github.io/type-effection-by-js",
  "https://imcuttle.github.io/walli-born",
  "https://imcuttle.github.io/week-record-2",
  "https://imcuttle.github.io/why-v8-so-fast",
  "https://imcuttle.github.io/wow-picidae-transformer-react-render-support-live-edit",
  "https://imcuttle.github.io/you-don%27t-know-console",
  "https://imcuttle.github.io/you-paint-i-guess-by-socketio",
  "https://imcuttle.github.io/在线编辑国际化文本",
  "https://imcuttle.github.io/毕业论文初稿",
  "https://imcuttle.github.io/毕设周记4",
  "https://imcuttle.github.io/",
  "https://imcuttle.github.io/posts/1",
  "https://imcuttle.github.io/posts/2",
  "https://imcuttle.github.io/posts/3",
  "https://imcuttle.github.io/posts/4",
  "https://imcuttle.github.io/posts/5",
  "https://imcuttle.github.io/posts/6",
  "https://imcuttle.github.io/htm/圣杯布局.html",
  "https://imcuttle.github.io/htm/双飞翼.html",
  "https://imcuttle.github.io/posts/7",
  "https://imcuttle.github.io/posts/8",
  "https://imcuttle.github.io/posts/9",
  "https://imcuttle.github.io/posts/10",
  "https://imcuttle.github.io/posts/11",
  "https://imcuttle.github.io/htm/bezier.html",
  "https://imcuttle.github.io/htm/4三维基本变换5组合变换.doc",
  "https://imcuttle.github.io/developers.google.com/web/updates/2015/12/background-sync",
  "https://imcuttle.github.io/值得细嚼",
  "https://imcuttle.github.io/免费证书",
  "https://imcuttle.github.io/2016/04/23/requestAnimationFrame_Vs_setInterval/",
  "https://imcuttle.github.io/htm/painter",
  "https://imcuttle.github.io/htm/canvas1.html",
  "https://imcuttle.github.io/THEME_README.md",
]
`;
